pyEpiabm/pyEpiabm/tests/test_sweep/test_host_progression_sweep.py

```diff
@@ -176,6 +189,85 @@ def test_icdf_exception_raise(self):
         test_sweep.transition_time_matrix.loc[row_index, column_index].\
             icdf_choose_noexp.assert_called_once()
 
+    def test_infectiousness_progression(self):
+        """Tests that the output is a numpy ndarray and that the tail of the
+        array is 0, starting at the element k.
+        """
+        # Parameters to determine where the tail starts
+        infectious_period = pe.Parameters.instance().asympt_infect_period
+        model_time_step = 1 / pe.Parameters.instance().time_steps_per_day
```

Comment:
If these scalars are just parameters, can we put the values in here rather than having to rely on their values in the parameter file. Will be especially pertinent once we are reading in parameters from an external file we don't want to be involved in the tests

Reply:
I think we would want these to be read from the parameter file (and from an external file if it becomes our way to input parameters), as these are the kind of parameters that users might want to change easily.  The model time step really need to stay linked to time steps per day, which is used elsewhere in the code.

Reply:
I think I need to call them from param, because these same parameters are called from the parameter file in the init of the host-progression-sweep, and the test would fail if the user changes the params.

Reply:
Yes absolutely in the main code, but best not to rely on an external module in the tests (particularly as it will soon import in external values) when you could simply copy the floats across here!

Reply:
I still think we need it, because if we change the parameters later on, then the test would fail because these two parameters would not match those that are taken in the main code.

Reply:
Now that I can easily mock the parameters instance everything is mocked in the test, so that should solve this.